docs/src/install/install.rst

```diff
@@ -44,13 +44,15 @@ After downloading the Opaque codebase, build and test it as follows.
    By default, Opaque runs in hardware mode (environment variable ``MODE=HARDWARE``).
    If you do not have a machine with a hardware enclave but still wish to test out Opaque's functionality locally, then set ``export MODE=SIMULATE``.
 
-3. There is an ``sbt`` task you can use to generate private and symmetric keys using OpenSSL for Opaque SQL.
+3. To generate new keys using OpenSSL, you can use the following ``sbt`` task:
 
    .. code-block:: bash
 
                   build/sbt keys
 
-   Alternatively, you can use your own keys, though this is generally not recommended.
+   *Note that, by default, Opaque SQL uses a pre-generated RSA private key for enclave signing. This key should be not be used in a production environment.* 
+
+   Alternatively, you can use your own keys, though this is generally not recommended. 
```

Comment:
The key that `PRIVATE_KEY_PATH` points to has to be identical to the one on the client, correct? If that's the case then I don't think we need this SBT task anymore.

Reply:
The client won't have the private key, but the client should have the public key corresponding to the key at `PRIVATE_KEY_PATH`. Why wouldn't we need the SBT task anymore? Doesn't it still generate the symmetric key?

Reply:
lol too many keys

Yeah it does, but that symmetric key will be useless after https://github.com/mc2-project/opaque-sql/pull/233 is merged in

Reply:
Sounds good 😄